<launch>
    <node pkg="lasr_vision_yolov8" type="service" name="yolo_service" output="screen"/>
    <node pkg="coffee_shop" type="tf_transform.py" name="tf_transform" output="screen"/>
    <node pkg="coffee_shop" type="test_people_pose.py" name="test_people_pose" output="screen"/>
    <node pkg="coffee_shop" type="test_people_pose_laser.py" name="people_pose_laser" output="screen"/>
</launch>
